src/components/Charts.tsx

```diff
@@ -4,22 +4,30 @@ import { LibraryData } from "../interfaces";
 
 export interface ChartsProps {
   data: {[key: string]: LibraryData[]};
+  width?: string | number;
+  height?: string | number;
 }
 
 export default class Charts extends React.Component<ChartsProps, {}> {
+  static defaultProps = {
+    width: "50%",
+    height: 400
```

Comment:
I don't know whether it's because of this, but...now the labels on the bar chart get smushed together (as per your second screenshot).

Reply:
Yes, but that's only if you try to squeeze the page down to a mobile view. Anything larger and the bars are evenly spaced. I don't think any one will really be viewing this on mobile anyway. So either the `ResponsiveContainer` can go since it doesn't add much and the props are incorrect, or deal with a squished mobile view.